sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -310,4 +315,22 @@ export interface ApplicationData {
    * Student number.
    */
   studentNumber?: string;
+  /**
+   * Program id selected by the student.
+   */
+  selectedProgram?: number;
+  /**
+   * Program name selected by the student.
+   * This is for html component of readonly form.
+   */
+  selectedProgramName?: string;
+  /**
+   * Location id selected by the student.
+   */
+  selectedLocation?: number;
+  /**
+   * Location name selected by the student.
+   * This is for html component of readonly form.
+   */
+  selectedLocationName?: string;
```

Comment:
Is this part of the dynamic data that is persisted to the jsonB column?

Reply:
this won't be saved in database, it's part of the initial data for read-only mode